### ISSUE TITLE
关于训练的LR问题

### ISSUE THREAD

**Issue by zcswdt** [PaddlePaddle/PaddleOCR#2120]
您好，有几个问题请教。

1.我看到2.0动态图这版和之前1版在训练的学习率调整有些不同，之前学习率采用了预热学习率的方式，2.0这版仿佛舍弃了预热过程。
2.在训练的过程中学习率为什么会在一个值上停更很久，如下图所示。
![image](https://user-images.githubusercontent.com/43515926/109244010-e61adb00-7818-11eb-973c-0cb9c8eaefed.png)

**Comment by LDOUBLEV** [comment]
warmup 是可以加上的，动态图代码也支持，可能有的配置文件忘了加了

> 在训练的过程中学习率为什么会在一个值上停更很久，如下图所示。

应该是你每个epoch的迭代次数太多

**Comment by zcswdt** [comment]
> warmup 是可以加上的，动态图代码也支持，可能有的配置文件忘了加了
> 
> > 在训练的过程中学习率为什么会在一个值上停更很久，如下图所示。
> 
> 应该是你每个epoch的迭代次数太多

谢谢回复，请教一下，学习率在每次迭代更新的时候，lr如何取值呢？我在函数中看到其中的cycle不是固定的值1么。https://github.com/PaddlePaddle/PaddleOCR/blob/3a11283588be5aa5b2b553d794d7f38230f9f984/ppocr/optimizer/lr_scheduler.py#L43
所以`reletive_epoch`是怎么更新的呢？谢谢
https://github.com/PaddlePaddle/PaddleOCR/blob/3a11283588be5aa5b2b553d794d7f38230f9f984/ppocr/optimizer/lr_scheduler.py#L46
